types/bmp-js/index.d.ts

```diff
@@ -0,0 +1,77 @@
+// Type definitions for bmp-js 0.1
+// Project: https://github.com/shaozilee/bmp-js
+// Definitions by: Robert Krahn <https://github.com/rksm>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.7
```

Comment:
I don’t see anything in here that requires TypeScript 3.7—you should remove this line.

Reply:
Done.